src/components/Modals/DeleteDocument/DeleteDocModal.jsx

```diff
@@ -32,11 +35,13 @@ function DeleteDocumentModal({ closeModal, currentDocIndex, documentsIds }) {
 
   return (
     <Modal onClick={closeModal}>
-      <div className="flex flex-col items-center">
-        <span className="text-xl p-10">
-          Are you sure you want to permanently delete this document?
-        </span>
-        <div className="flex justify-evenly w-full">
+      <ContentWrapper>
+        <Content>
+          <Message>
+            Are you sure you want to permanently delete this document?
+          </Message>
+        </Content>
+        <div className="flex justify-evenly items-center  w-full">
```

Comment:
whitespace 제거 부탁드립니다 👁️

```suggestion
        <div className="flex justify-evenly items-center w-full">
```